### ISSUE TITLE
PAL loss question

### ISSUE THREAD

**Issue by jyang68sh** [tianyu0207/PEBAL#10]
Hi

First of all, great paper. 

In the paper:

"The minimisation of the loss in (3) will abstain from classifying outlier
pixels into one of the inlier classes, where a pixel is estimated to be an outlier
with aω."

"ℓpal learns to abstain the prediction of outlier pixels from x out containing OE objects (i.e.,
cyan coloured masks) and calibrate the logit of inlier classes (i.e., reduction of the inlier logits) from both inlier image x
in and outlier image xout"

I dont quite understand these two statements.

1. Min_Pal is achieved when a pixel is estimated to be an outlier class
2. if 1 is correct, how ℓpal is helping to abstain the prediction of outlier pixel. From what I understood, we are trying to minimize the PAL loss when model training, minimizing is pushing a pixel to be estimated to be an outlier.

This is confusing to me. Hope somebody could explain. 

My key point is your statements is reasonable and correct. I just didnt get the how it works.

**Comment by yyliu01** [comment]
In short, adding the confidence of the external channel back to the original channels can indirectly lead the model to be aware of the anomalous input data.  The cross-entropy will only penalise the logits that are related to the one-hot target, for the predictions of the anomalies, the model will tend to put all the confidence into the external one, which will always be correct (and avoid the penalties, or **minimize the PAL loss**). 
 
This part is highly based on the gambler loss (https://arxiv.org/abs/1907.00208), please take a look.
